packages/cli/lib/base-generator.js

```diff
@@ -129,6 +129,10 @@ module.exports = class BaseGenerator extends Generator {
         'Result:',
         jsonStr === undefined ? '(undefined)' : JSON.stringify(jsonStr),
       );
+      if (jsonStr === '') {
+        debug('Received an empty string, returning back empty config: {}');
+        return {};
+      }
```

Comment:
Do we want to add a test to verify this fix? Such test would have to spawn a new child process to invoke the generator. That seems like a too much effort to me, considering that other parts of `_readJSONFromStdin` are not covered by tests either, at least AFAIK.